Example/Tests/BPKBadgeSnapshotTest.m

```diff
@@ -0,0 +1,62 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
+#import <Backpack/Badge.h>
+
+@interface BPKBadgeSnapshotTest : FBSnapshotTestCase
+
+@end
+
+@implementation BPKBadgeSnapshotTest
+
+- (void)setUp {
+    [super setUp];
+    self.recordMode = NO;
+}
+
+- (UIStackView *)getStackView {
+    UIStackView *stackView = [[UIStackView alloc] initWithFrame:CGRectMake(0, 0, 300, 300)];
+    stackView.axis = UILayoutConstraintAxisVertical;
+    stackView.distribution = UIStackViewDistributionEqualCentering;
+    return stackView;
+}
+
+- (void)testViewSnapshotWithTypes {
```

Comment:
Can we make this not stretch the badges to the full width of the stack view? It's not really how they end up used most of the time